extensions/notification/NotificationQueuePopulator.js

```diff
@@ -125,8 +125,14 @@ class NotificationQueuePopulator extends QueuePopulatorExtension {
                 key,
                 eventType,
             });
+            let wasPushedToSharedTopic = false;
             // validate and push kafka message foreach destination topic
             this.notificationConfig.destinations.forEach(destination => {
+                // avoid pushing the same message multiple times to the
+                // internal shared topic
```

Comment:
This may be a partial fix: eg if the same "external" topic is specified in multiple targets' config
→ either we need to handle that, or to detect/prevent (or at least document) this case/limitation

Reply:
that's easy to handle, better do that